src/js/ripple/remote.js

```diff
@@ -519,16 +519,16 @@ Remote.prototype._handleMessage = function(message, server) {
     message = JSON.parse(message);
 
   /*eslint-disable no-empty*/
-  } catch (e) {
-    // TODO: why are we suppressing this?
-    console.error(e);
-  }
+  } catch (e) { }
```

Comment:
I don't think it matters to whether JSON parsing fails if the error message is descriptive. I'm sympathetic to only handling object-type messages on Remote